helm-chart/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: safe-config-service-chart
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```

Comment:
Could this be made to reflect the real app version? eg v2.2.0  for the latest release tag that exists?

Reply:
That's the release version of the original `gnosis/safe-config-service` right? Are we gonna use the same versioning in this fork?

Reply:
Thanks for the added context. Good question. Unless @dckesler or @nicolasbrugneaux have an opinion, we can keep this as is for now until we reach a new decision.

Reply:
I think we're happy to defer to your best judgement here. I think once we start making versions of the celo-safe, we'll try to keep it in sync with the helm's version

Reply:
Thanks @nicolasbrugneaux !! I guess we can leave this as a placeholder temporarily as it will be overridden by `.Values.image.tag` in the `values.yaml` anyway.